script.js

```diff
@@ -9,3 +9,20 @@
 // Add a `facial-hair` property to `instructor`.
 
 // Log the value of the `facial-hair` property to the console.
+var colors = ["red", "yellow", "green"];
+var colorsLength = colors.length;
+console.log(colors[2])
+
+for (i= 0; i < [3]; i++) {
```

Comment:
This technically works but you should not wrap `3` in brackets. Whatever is to the right of, in this case, `<` should evaluate to a number -- not an array.